hw/net/rocker/rocker_hw.h

```diff
@@ -475,6 +504,16 @@ enum rocker_of_dpa_overlay_type {
     (ROCKER_GROUP_TYPE_SET(ROCKER_OF_DPA_GROUP_TYPE_L3_UCAST) |\
      ROCKER_GROUP_INDEX_LONG_SET(index))
 
+/* cmd info (nested tlvs) for P4 */
+enum {
+    ROCKER_TLV_P4_RMT_INFO_UNSPEC = 0,  /* don't use 0 for tlv type */
```

Comment:
Ok, maybe INFO is best since that's the nest it's under.  Again, of-dpa will be an exception, but maybe we can fix of-dpa to be consistent once the dust settles.